packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -33,6 +33,15 @@ export class FeedbackForm extends React.Component<FeedbackFormProps, FeedbackFor
     ...defaultConfiguration
   }
 
+  private static _savedState: FeedbackFormState = {
```

Comment:
The `isVisible` doesn't seem to be used, we can omit it like this `Omit<FeedbackFormState, 'isVisible'>`.

Reply:
Good point 👍 
Updated with 1efb23e290f3aeef6977e8e714d1c191a1e8342c